solutions/tests/test_two_sum.py

```diff
@@ -0,0 +1,51 @@
+"""
+Unit tests for the two_sum function.
+"""
+
+import unittest
+
+from solutions.two_sum import two_sum
+
+
+class TestTwoSum(unittest.TestCase):
+    """
+    Unit tests for the two_sum function.
+    """
+
+
+def test_empty_or_single_element_list(self):
+    with self.assertRaises(ValueError):
+        two_sum([], 9)
+    with self.assertRaises(ValueError):
+        two_sum([5], 5)
+
```

Comment:
here if you can add docstring for this test and move it to bottom of tests would be more readable